themes/default/content/blog/meals-for-ukraine/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Teaming up to help deliver more meals to Ukraine"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-05-16T09:00:54+02:00
+meta_desc: Buoyant, Civo, Cockroach Labs, Fairwinds, Pulumi, & Jetstack team up to donate to World Central Kitchen to provide meals in response to humanitarian crises.
+meta_image: kubecon-ukraine.png
+authors:
+    - wendy-smith
+tags:
+    - community
+    - pulumi-events
+---
+Why not do good while learning about cloud native tech? [Buoyant](https://buoyant.io/), [Civo](https://www.civo.com/), [Cockroach Labs](https://www.cockroachlabs.com/), [Fairwinds](https://www.fairwinds.com/), [Pulumi](https://www.pulumi.com/), and [Jetstack](https://www.jetstack.io/) have teamed up to donate $2 for each virtual booth visitor to [World Central Kitchen](https://wck.org/) (WCK), a non-profit committed to providing meals in response to humanitarian crises.
+
+<!--more-->
+Working with restaurants across Ukraine and in six countries welcoming refugees, WCK is providing hundreds of thousands of daily meals for people in need. WCK serves meals at shelters, hospitals, transit hubs, and more. It's a wonderful mission and, if you agree, we encourage you to [donate to WCK directly](https://donate.wck.org/give/398293#!/donation/checkout) 💙 🇺🇦 💛
+
+## Tweets
+
+{{< tweet id="1510622155775619078" user="pulumipus" >}}
```

Comment:
I'm not sure what difference it makes, but I believe this should be the tweet author:

```suggestion
{{< tweet id="1510622155775619078" user="WCKitchen" >}}
```